example/lib/main.dart

```diff
@@ -29,17 +28,17 @@ class _MyAppState extends State<InitialScreen> {
   // Platform messages are asynchronous, so we initialize in an async method.
   Future<void> initPlatformState() async {
     await Purchases.setDebugLogsEnabled(true);
-    await Purchases.setup("api_key");
+    const apiKey = String.fromEnvironment("API_KEY", defaultValue: null);
```

Comment:
I wish getting env variables was this easy everywhere.

Reply:
does this mean that from now on, for testing, we need to replace the default value here?

Reply:
Well, you know that it only works if it's a const? haha 

Also, yes, the defaultValue should be replace for testing. Should I add a comment?

Reply:
🤔 it might lead to confusing situations if customers already have `API_KEY` set to something. 
maybe `REVENUECAT_API_KEY`?